frontend/src/views/web/change-password.html

```diff
@@ -4,27 +4,35 @@
             <div class=" ev-card-panel ev-z-depth-5">
                 <div class="ev-md-container ev-panel-title"><strong>Change Password</strong></div>
                 <div class="ev-md-container ev-card-body change-passowrd-card">
-                    <form ng-submit="changepwd.changePassword()">
+                    <form name="changepwdForm" ng-submit="changepwd.changePassword(changepwdForm.$valid)" novalidate>
 
                         <div class="input-field align-left">
-                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" focus-if>
+                            <input id="old_password" type="password" class="validate text-dark-black dark-autofill w-400" ng-model="changepwd.user.old_password" required focus-if>
```

Comment:
I think now your confusion is gone. :smile: 
I have a help link for you regarding `ng-message`.
[Get Started with ng-message](https://scotch.io/tutorials/angularjs-form-validation-with-ngmessages)